versioned_docs/version-2.0.0/quickstart/node-express-mongoose.md

```diff
@@ -45,31 +42,19 @@ npm install
 
 ## Installation 📥
```

Comment:
the structure of docs would be in :- https://github.com/keploy/docs/blob/main/versioned_docs/version-2.0.0/quickstart/python-django-sql.md